tests/PHPUnit/Unit/SecurityPolicyTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tests\Unit;
+
+use Piwik\View\SecurityPolicy;
+use Piwik\Config;
+
+/**
+ * @group Core
+ * @group SettingsServer
+ */
+class SecurityPolicyTest extends \PHPUnit\Framework\TestCase
+{
+    private $securityPolicy;
+    private $defaultPolicy = "default-src 'self' 'unsafe-inline' 'unsafe-eval'; ";
+    private $generalConfig;
+
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        // set the default config explicitly so tests can change them without needing to reset them
+        $this->generalConfig =& Config::getInstance()->General;
+        $this->generalConfig['csp_enabled'] = 1;
+        $this->generalConfig['csp_report_only'] = 1;
+
+        $this->securityPolicy = new SecurityPolicy();
```

Comment:
Due to the DI change this now needs to be `new SecurityPolicy(Config::getInstance())` or `StaticContainer::get(SecurityPolicy::class)`

Reply:
True thanks.